### DISCUSSION TITLE
Would it make sense to show the (ssh) server ip instead of 127.0.0.1 in the overview for ssh-tunneled connections?

### DISCUSSION THREAD

**Discussion by PureIncompetence** [gujjwal00/avnc#123] [Ideas]
Right now the shown host IP is always the one of the vnc-server (usually 127.0.0.1 for ssh-tunneled connections), which technically is undoubtly correct, but imho kind of unintuitive, as I would expect to see the the remote host ip (no matter if direct vnc or vnc through ssh) I'm connecting to. Additionally it would create a better distinction between different hosts than showing localhost for every ssh-tunneled connection.

**Comment by gujjwal00** [comment]
Seems like a nice improvement.
Thanks for the suggestion.

**Comment by gujjwal00** [comment]
Implemented in https://github.com/gujjwal00/avnc/commit/7ef81be40648b9ee3114ade27f606e02043d331f

**Reply by PureIncompetence** [reply]
Thank you, very much appreciated.
